wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -75,19 +56,13 @@ class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
             } else if userClient.isLegalHoldDevice {
                 nameLabel.text = L10n.Localizable.Device.Class.legalhold
             }
-
+            let proteusId = userClient.displayIdentifier.fingerprintStringWithSpaces.uppercased()
+            proteusIdLabel.text =  deviceDetailsSection.Proteus.id + ": " + proteusId
```

Comment:
**suggestion**: use it's own localized string with parameter. I am afraid this could be an issue with right to left languages